### ISSUE TITLE
Margin?

### ISSUE THREAD

**Issue by derolly** [AndreasReitberger/3D-Print-Cost-Calculator#12]
Hello,

no matter what I do, your app does not calculate any margin for my projects, no matter what settings I use.
Is this a known bug? or am I missing something?

**Comment by AndreasReitberger** [comment]
Hi,
what version are you using? I test the latest version and there, it is working for me.
Can you share some screenshots of your settings, so that I can reproduce your state on my end? I'll check ith then :)

**Comment by derolly** [comment]
Im using the latest version 1.4.8.0.
What Screenshots do you need? The margin calculation simply does not appear anywhere and the percentage slider has no effect on the calculation whatsoever. I also reinstalled the app and deleted the C:\...\Roaming\PrintCostCalculator3d folder

![Unbenannt-1](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/0a379209-0742-4c73-8eeb-9e26b04bbfe7)
![Unbenannt-2](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/f52a365f-1798-461c-b7cb-84ff9749f03d)
![Unbenannt-3](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/1bf362d9-9e5e-4769-b872-f2ea8d961358)
![Unbenannt-4](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/7d13a0d8-b75a-4dc4-9521-c522b8d6c9f6)
![Unbenannt-5](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/d0c0889e-0465-4c8e-9d08-14b9f0d6b072)
![Unbenannt-6](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/e743bfa8-32ab-4606-bc93-468a94682600)
![Unbenannt-7](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/d2f790e3-8241-4d6c-bb8c-4ee9cff70ec0)
![Unbenannt-8](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/dfd63f8d-affb-4d3e-a6c0-04b7b631dd36)
![Unbenannt-9](https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/assets/31779133/c9eea147-93be-4248-afd2-8ddada64909a)

**Comment by AndreasReitberger** [comment]
Hi, thanks for sharing the screenshots.
I just made a calculation with the same setup and it is working on my end.

Can you share your settings file and database with me?
See how to do so here:

- DB: https://ar3dp.de/docs/3d-druckkosten-kalkulator/einstellungen/datenbank/
- Settings: https://ar3dp.de/docs/3d-druckkosten-kalkulator/einstellungen/einstellungsdatei-anzeigen/

Please do this in a private email to: [kontakt@andreas-reitberger.de](mailto:kontakt@andreas-reitberger.de)
I will investigate this issue then directly with your data.

Thank you!

**Comment by AndreasReitberger** [comment]
@derolly
I was able to reproduce the issue with your data and the steps you provided.
The fix is already done and will be tested on the application level now.

The fix will be released with the upcoming update `1.5.0-beta`. I'll inform you here once the release is done.

Fix in library:
https://github.com/AndreasReitberger/3dPrintCostCalculatorSharp/pull/131

Thank you for reporting this!

**Comment by derolly** [comment]
Nice to hear that I could help!

**Comment by AndreasReitberger** [comment]
@derolly 
Would you mind if you tested the latest release?
https://github.com/AndreasReitberger/3D-Print-Cost-Calculator/releases/tag/v1.5.0-beta

Thank you!

**Comment by derolly** [comment]
I tested your update and it seems to work for me, had no problems! Thanks for your work!

**Comment by AndreasReitberger** [comment]
Awesome, thank you very much. Then I'll close this issue!
